### PR TITLE
DFR-1494: Missing name from hearing notice although added to the case…

### PR DESCRIPTION
**Pull request by srochani**
… and on other orders

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by rishikrsharma — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ApprovedOrderNoticeOfHearingService.java [hunk 1]
```diff
@@ -179,7 +179,11 @@ private Map<String, Object> addValuesToPlaceHoldersMap(CaseDetails caseDetails,
         placeholdersMap.put("CCDCaseNumber", caseDetails.getId());
         placeholdersMap.put("DivorceCaseNumber", nullToEmpty(caseDetails.getData().get(DIVORCE_CASE_NUMBER)));
         placeholdersMap.put("ApplicantName", documentHelper.getApplicantFullName(caseDetails));
-        placeholdersMap.put("RespondentName", documentHelper.getRespondentFullNameContested(caseDetails));
+        if (caseDataService.isConsentedApplication(caseDetails)) {
```
Comment (EllisD-B) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ApprovedOrderNoticeOfHearingService.java:182]: This logic can be simplified - there's a method in the CaseDataService called buildFullRespondentName which does the consented/contested check for you

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ConsentHearingService.java [hunk 2]
```diff
@@ -136,7 +136,11 @@ private ConsentedHearingDataWrapper generateHearingDocument(ConsentedHearingData
         caseData.put("ccdCaseNumber", caseDetailsCopy.getId());
         caseData.put("courtDetails", buildFrcCourtDetails(hearingCaseData));
         caseData.put("applicantName", documentHelper.getApplicantFullName(caseDetailsCopy));
-        caseData.put("respondentName", documentHelper.getRespondentFullNameContested(caseDetailsCopy));
+        if (caseDataService.isConsentedApplication(caseDetails)) {
```
Comment (EllisD-B) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ConsentHearingService.java:139]: same as above

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ApprovedOrderNoticeOfHearingService.java (hunk 1 above)
Comment (srochani) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ApprovedOrderNoticeOfHearingService.java]: Thanks, updated

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ConsentHearingService.java (hunk 2 above)
Comment (srochani) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ConsentHearingService.java]: Thanks, updated

**Review by EllisD-B — APPROVED** [review]

**Pull request merged**